core/src/js/components/mercenaries/desktop/mercenaries-meta-hero-stats.component.ts

```diff
@@ -31,12 +31,18 @@ const THRESHOLD = 50;
 		`../../../../css/component/mercenaries/desktop/mercenaries-meta-hero-stats.component.scss`,
 	],
 	template: `
-		<div class="mercenaries-stats" scrollable>
+		<div class="mercenaries-stats" *ngIf="(stats$ | async).length; else searchEmptyState" scrollable>
```

Comment:
I think it is good practice to use the optional chaining, like `(stats$ | async)?.length` to avoid breaking everything in case stats$ is null